files/image_config/interfaces/interfaces-config.sh

```diff
@@ -1,6 +1,4 @@
 #!/bin/bash
 
 sonic-cfggen -m /etc/sonic/minigraph.xml -t /usr/share/sonic/templates/interfaces.j2 >/etc/network/interfaces
-ifdown eth0 && ifup eth0
-ifdown lo && ifup lo
-
+service networking restart
```

Comment:
i think they are here doing the same thing. is `systemctl` newer than `service` command? I could switch to `systemctl`.